### PR TITLE
"Wait" and "is not present" steps are added

### PR DESCRIPTION
**Pull request by Lisalexandr**
Two steps are added:
- Wait for time in milliseconds
- Element should not be present

### CODE REVIEW

File: src/helpers/objects.processor.js [hunk 1]
```diff
@@ -39,6 +39,17 @@ function injectInto(locator, injection) {
     return locator + '[' + injection;
 }
 
+/**
```
Comment (alexkrechik) [inline src/helpers/objects.processor.js]: This function should be moved to the utils (see _r for example). Only Object processing methods should be placed here

File: src/steps/then.js [hunk 2]
```diff
@@ -16,6 +17,14 @@ module.exports = function () {
...
             browser.$(locator).waitForExist();
         });
+        Then(_r(`^${pageObject} should not be present$`), (object) => {
+            /**
+             * The element should not be present
+             * @type {PageObject}
+             */
+            const locator = getPageObject(object);
 
+            browser.$(locator).waitForExist(timeout, true);
```
Comment (alexkrechik) [inline src/steps/then.js]: We should use default timeout here. Look at the waitForExist realization.

File: src/steps/when.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+module.exports = function () {
+    defineSupportCode(({ When }) => {
+
+        When(_r(`I wait ${dictionaryObject} ms$`), (timeValue) => {
+            /**
+             * Wait in milliseconds
+             * @type {String} or {DictionaryObject}
+             */
+            const time = getInteger.call(this, timeValue);
```
Comment (alexkrechik) [inline src/steps/when.js]: We should use getDictionaryObject() result, not the got value due to timeValue could be something like `"dictionary"."object"`.

**Comment by Marketionist** [comment]
Fixes https://github.com/revjet-qa/wdio-cucumber-steps/issues/37

**Pull request merged**